allennlp/data/dataset_readers/dataset_reader.py

```diff
@@ -0,0 +1,21 @@
+from allennlp.data import Dataset
```

Comment:
Is this file in here by accident?  Don't we have this at `allennlp/data/dataset_reader.py`?